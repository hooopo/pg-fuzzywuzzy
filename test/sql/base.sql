\set ECHO none
BEGIN;
\i sql/fuzzywuzzy.sql
\set ECHO all

-- You should write your tests

SELECT ratio('foo', 'bar');
SELECT ratio('foo', 'foo');
SELECT ratio('你好', '你好世界');
SELECT ratio('hello', 'hello world');
SELECT ratio('', '');
SELECT ratio(null, 'hello');
SELECT ratio(null, null);

ROLLBACK;
