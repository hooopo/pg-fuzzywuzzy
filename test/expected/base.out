\set ECHO none
-- You should write your tests
SELECT ratio('foo', 'bar');
 ratio 
-------
     0
(1 row)

SELECT ratio('foo', 'foo');
 ratio 
-------
   100
(1 row)

SELECT ratio('你好', '你好世界');
 ratio 
-------
    67
(1 row)

SELECT ratio('hello', 'hello world');
 ratio 
-------
    63
(1 row)

SELECT ratio('', '');
 ratio 
-------
     0
(1 row)

SELECT ratio(null, 'hello');
 ratio 
-------
     0
(1 row)

SELECT ratio(null, null);
 ratio 
-------
     0
(1 row)

ROLLBACK;
